core/domain/suggestion_registry_test.py

```diff
@@ -2016,9 +2165,14 @@ def test_pre_update_validate_complains_if_nothing_changed(self):
             'exploration.exp1.thread1', 'exp1', 1,
             suggestion_models.STATUS_ACCEPTED, self.author_id,
             self.reviewer_id, change,
-            'question.topic_1', 'en', self.fake_date)
-
-        new_change = {
+            'question.topic_1', 'en', False, self.fake_date)
+
+        new_change: Dict[
```

Comment:
If this is used quite a lot, should we move it to a `TypeVar`?

Reply:
Done!